modules/nw-rfhe-installing-operator-cli.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * monitoring/using-rfhe.adoc
+
+:_content-type: PROCEDURE
+[id="nw-rfhe-installing-operator-cli_{context}"]
+= Installing the Baremetal Hardware Event Proxy Operator using the CLI
+
+As a cluster administrator, you can install the Operator by using the CLI.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware with nodes that have Redfish hardware.
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the Baremetal Hardware Event Proxy Operator.
+
+.. Save the following YAML in the `hw-events-namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-hw-events
+  annotations:
+    workload.openshift.io/allowed: management
+  labels:
+    name: openshift-hw-events
+    openshift.io/cluster-monitoring: "true"
+----
+
+.. Create the `Namespace` CR:
++
+[source,terminal]
+----
+$ oc create -f hw-events-namespace.yaml
+----
+
+. Create an Operator group for the Baremetal Hardware Event Proxy Operator.
+
+.. Save the following YAML in the `hw-events-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: baremetal-hardware-event-proxy-operator-group
+  namespace: openshift-hw-events
+spec:
+  targetNamespaces:
+  - openshift-hw-events
+----
+
+.. Create the `OperatorGroup` CR:
++
+[source,terminal]
+----
+$ oc create -f hw-events-operatorgroup.yaml
+----
+
+. Subscribe to the Baremetal Hardware Event Proxy Operator.
+
+.. Save the following YAML in the `hw-events-sub.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: baremetal-hardware-event-proxy-operator-subscription
+  namespace: openshift-hw-events
+spec:
+  channel: "stable"
+  name: baremetal-hardware-event-proxy-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+.. Create the `Subscription` CR:
++
+[source,terminal]
+----
+$ oc create -f hw-events-sub.yaml
+----
+
+.Verification
+
+To verify that the Operator is installed, run the following command:
+
+[source,terminal]
+----
+$ oc get csv -n openshift-hw-events -o custom-columns=Name:.metadata.name,Phase:.status.phase
+----
+
+.Example output
+[source,terminal]
+----
+Name                         Phase
+4.10.0-202201261535          Succeeded
```

Comment:
Please change "4.10.0-202201261535" to "baremetal-hardware-event-proxy-operator.v4.10.0".

e.g., actual output is like this:
```
[kni@registry ~]$ oc get csv -n openshift-hw-events -o custom-columns=Name:.metadata.name,Phase:.status.phase
Name                                              Phase
baremetal-hardware-event-proxy-operator.v4.10.0   Succeeded
```